src/tabbar.cpp

```diff
@@ -127,6 +129,24 @@ void TabBar::setNewTabButton()
     setTabButton(idx, QTabBar::RightSide, Q_NULLPTR);
 }
 
+void TabBar::moveToNextTab()
+{
+    auto index = currentIndex() + 1;
+    if (index == count() - 1) {
+        index = 0; // if we are on last tab, move to first tab (library).
+    }
+    setCurrentIndex(index);
+}
```

Comment:
If I understand correctly, this jumps from the last-but-one tab to the first tab (skips the last tab).

Reply:
Yes, the last tab in TabBar is "+" icon

Reply:
Then make it explicit (with the help of a comment)

Reply:
A more explicit way of addressing this subtlety (instead of having to mention and explain it at every location where it matters) is to introduce a function `realTabCount()` and document the issue in only one place

Reply:
Done, thanks!